src/components/App.js

```diff
@@ -1,19 +1,118 @@
 import React from 'react';
+import MenuComponent from "./MenuComponent";
+import Basket from './Basket';
 
 import '../styles/App.scss';
 
+
 class App extends React.Component {
-  constructor(){
+  constructor() {
     super();
+
+    this.state = {
+      menu : [],
+      currentOrder: [],
+      totalOrder: [],
+      totalPrice: 0,
+      showOrder: false
+    };
+    this.getFetch = this.getFetch.bind(this);
+    this.addOrder = this.addOrder.bind(this);
+    this.removeOrder = this.removeOrder.bind(this);
   }
 
-  render(){
-    return (
-      <div>
-        Delivereat app
-      </div>
-    )
+
+  componentDidMount() {
+    this.getFetch();
+  }
+
+
+  getFetch() {
+    const serverFetch = `http://localhost:8080/menu`
+
+    fetch(serverFetch)
+    .then(response => response.json())
+    .then(content => {
+      console.log("menu content",content)
+      this.setState( {
+        menu : content
+       } )
+     }
+    );
+  }
+
+  addOrder(order) {
+   const newTotalOrder = [...this.state.totalOrder, order];
+   const newTotalPrice = Number(this.state.totalPrice) + order.price;
+   console.log("newTotalPrice",newTotalPrice);
+   this.setState( {
+     currentOrder : order,
```

Comment:
It's not clear how currentOrder is used. It is initialised as an array, but get's an object set in it. Also it looks like `currentOrder` is not used in the application